### PR TITLE
Updated Dockerfile for r-image

### PR DESCRIPTION
**Pull request by jpilorget**
Simplified Jupyter image with no pyOpenSSL issues or conda dependencies like the previous one.

*Issue #, if available:* solves issue #21

*Description of changes:* Changed the base Dockerfile using the one in https://hub.docker.com/r/jupyter/minimal-notebook as the base image (same approach as previous R image).

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: examples/r-image/Dockerfile [hunk 1]
```diff
@@ -1,111 +1,66 @@
-# This project is licensed under the terms of the Modified BSD License (also known as New or Revised or 3-Clause BSD), as follows:
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
 
-#    Copyright (c) 2001-2015, IPython Development Team
-#    Copyright (c) 2015-, Jupyter Development Team
+ARG OWNER=jupyter
+ARG BASE_CONTAINER=$OWNER/minimal-notebook
+FROM $BASE_CONTAINER
```
Comment (knaresh) [inline examples/r-image/Dockerfile:6]: why update the base image? Can you update the commit to summarize changes including why you had to change the base image?

File: examples/r-image/Dockerfile [hunk 2]
```diff
@@ -1,111 +1,66 @@
...
-
-# Neither the name of the Jupyter Development Team nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission.
-
-# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-FROM python:3.6
-
-ARG NB_USER="sagemaker-user"
-ARG NB_UID="1000"
-ARG NB_GID="100"
```
Comment (knaresh) [inline examples/r-image/Dockerfile:21]: Are these not working?

**Review by knaresh — COMMENTED** [review]
Can you also update what kind of testing you did?

File: examples/r-image/Dockerfile (hunk 1 above)
Comment (jpilorget) [reply examples/r-image/Dockerfile:6]: The base image of Python 3.6 had an issue with PyOpenSSL that made it impossible to build. By updating the base image we can rely on a managed image (minimal-notebook) hence reducing the risk of the base image having conflicts with the required packages.

File: examples/r-image/Dockerfile (hunk 2 above)
Comment (jpilorget) [reply examples/r-image/Dockerfile:21]: They are, I need to update this if we want to define the non-privileged user. Will do.

**Comment by marti-jorda-roca** [comment]
Hi, I have the same problem with the image and the new one is created correctly. The only problem is that when I try to execute any cell in Studio it just stays computing forever. Any update for this topic?

File: examples/r-image/Dockerfile (hunk 1 above)
Comment (jpilorget) [reply examples/r-image/Dockerfile:5]: Hi @knaresh please review if possible the new commit.

File: examples/r-image/Dockerfile (hunk 2 above)
Comment (jpilorget) [reply examples/r-image/Dockerfile]: Hi @knaresh please review if possible the new commit. This was solved.

**Comment by jpilorget** [comment]
> Hi, I have the same problem with the image and the new one is created correctly. The only problem is that when I try to execute any cell in Studio it just stays computing forever. Any update for this topic?

Hi @marti-jorda-roca this was solved with new commit. Please use and test as needed.

**Comment by tonyszhang** [comment]
Thank you for doing this. @jpilorget

**Review by knaresh — APPROVED** [review]

**Pull request merged**